.github/workflows/build.yml

```diff
@@ -219,30 +221,55 @@ jobs:
 
       - name: Login to Amazon ECR
         if: github.ref == matrix.app_env.branch
-        id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
 
-      - name: Build, tag, and push image to Amazon ECR
+      - name: Login to Amazon Public ECR
         if: github.ref == matrix.app_env.branch
-        id: build-image
-        env:
-          APP_ECR_REPOSITORY: twilreapi
-          NGINX_ECR_REPOSITORY: twilreapi-nginx
-          IMAGE_TAG: ${{ github.sha }}
-          AWS_REGION: us-east-1
-        run: |
-          cp -R public/* docker/nginx/public
+        run: aws ecr-public get-login-password | docker login --username AWS --password-stdin public.ecr.aws
+
+      - name: Build and push to AWS ECR `twilreapi`
+        id: docker_build_twilreapi
+        if: github.ref == matrix.app_env.branch
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: somleng/twilreapi:latest, somleng/twilreapi:${{ env.IMAGE_TAG  }}
+          cache-from: type=local,src=/tmp/.buildx-cache/buildx-cache-twilreapi
+          cache-to: type=local,dest=/tmp/.buildx-cache-twilreapi
```

Comment:
This won't work because we use assume role. I don't think we need this action, just use docker login normally

Reply:
Nope, it works just fine with assume role. For credentials, we either use this action or execute docker login directly from what we used to have.